x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/pagerduty.tsx

```diff
@@ -171,8 +176,8 @@ const PagerDutyParamsFields: React.FunctionComponent<ActionParamsProps<PagerDuty
     group,
   } = actionParams;
   const severityOptions = [
-    { value: 'critical', text: 'Critical' },
     { value: 'info', text: 'Info' },
+    { value: 'critical', text: 'Critical' },
```

Comment:
Not related to this PR but do we have an issue opened to use i18n for the `text` values of `severityOptions` and `eventActionOptions`?

Reply:
Added it here